pkg/verify/verify_test.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package app
+package verify
```

Comment:
I think we should leave this test where it is, and add additional tests that just test the verify functions. This test also tests CLI input and output

the unit test shouldn’t need to spin up a test server

Reply:
Actually, just realized this test was just for verifyHashedMessage, and we already have tests for the verify CLI. 
I’m good to move these, but can we implement it without needing a test server?

Reply:
In my opinion, running a full e2e call from a call to the verification step is ideal here. To be sure, nothing is broken from the timestamp action till the verification.

>  And also add test coverage for all functions?

Sounds good! I changed the tests to validate the main function.